### PR TITLE
Update deep-linking.md

### PR DESCRIPTION
**Pull request by chunhtai**
_Description of what this PR is changing or adding, and why:_

_Issues fixed by this PR (if any):_ Fixes #ISSUE-NUMBER

## Presubmit checklist
- [ ] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [ ] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [ ] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Review by sfshaza2 — APPROVED** [review]
lgtm

File: src/development/ui/navigation/deep-linking.md [hunk 1]
```diff
@@ -55,7 +55,18 @@ host name matches the name defined in `AndroidManifest.xml`:
 ```
 adb shell am start -a android.intent.action.VIEW \
     -c android.intent.category.BROWSABLE \
-    -d "http://flutterbooksample.com/book/1"
+    -d "http://flutterbooksample.com/book/1" \
+    <package name>
+```
+
+Replace the `<package name>` with the package name of your android app. If the package
+name is `com.example.myflutterapp`, the command will then be:
```
Comment (atsansone) [inline src/development/ui/navigation/deep-linking.md:63]: **issue:** Need to capitalize Android and use active voice in the second sentence.
```suggestion
Replace the `<package name>` with the package name of your Android app.
If you named the package `com.example.myflutterapp`, run the following command:
```

**Review by atsansone — CHANGES_REQUESTED** [review]
@chunhtai : Two small issues, then it should be ready to merge.

**Comment by chunhtai** [comment]
> @chunhtai : Two small issues, then it should be ready to merge.

Thanks I committed your suggestions.

**Review by atsansone — APPROVED** [review]

**Pull request merged**